src/test/resources/org/openstreetmap/atlas/geography/atlas/command/continent_map.cpg

```diff
@@ -0,0 +1 @@
+UTF-8
```

Comment:
Can we remove all the shapefiles? I think the text boundary below should be enough..